authentication/LegacyAuthQuickstart/AuthenticationExample/MainViewController.m

```diff
@@ -1167,18 +1167,19 @@ - (NSString *)currentNonce {
 - (void)startSignInWithGoogleFlow {
   // [START headless_google_auth]
   GIDConfiguration *config = [[GIDConfiguration alloc] initWithClientID:[FIRApp defaultApp].options.clientID];
+  [GIDSignIn.sharedInstance setConfiguration:config];
 
   __weak __auto_type weakSelf = self;
```

Comment:
optional nit, out of scope for these changes: This variable can be `__unsafe_unretained` like its Swift equivalent.

Reply:
Will disregard, since this is a legacy file only used for zip artifact build testing.